bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBS/VVjj_semileptonic/aQGC/aQGC_WMhadZlepJJ_EWK_LO_NPle1_mjj100pt10/aQGC_WMhadZlepJJ_EWK_LO_NPle1_mjj100pt10_madspin_card.dat

```diff
@@ -0,0 +1,10 @@
+set ms_dir ./madspingrid
+set BW_cut 15 # cut on how far the particle can be off-shell
+set max_running_process 1
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```

Comment:
You don't need to redefine the multiparticles in MadSpin as they are inherited from MadGraph (proc card).